docs/summary.rst.patch

```diff
@@ -0,0 +1,10 @@
+55,58d54
```

Comment:
Should this be included?

Reply:
I don't think so, because the summary page is already part of Read The Docs, so it seems it would be kind of circular.  Do you feel it makes sense to include it there?

Reply:
I forgot that there is the automatic check to have the README and Summary page stay in sync (https://github.com/horovod/horovod/blob/master/docs/summary.rst.patch)

I agree that this content would be an exception and wouldn't need to be exactly the same between the two pages, but think the automatic sync check is useful in general, so maybe best to leave it in for now.

Reply:
Agreed, definitely want to keep the check.  This just adds these lines as exceptions.